.gitignore

```diff
@@ -5,3 +5,4 @@ Manifest.toml
 *.jl.cov
 *.jl.*.cov
 *.jl.mem
+/usr
```

Comment:
What writes to `./usr`? Usually such things go in `./deps/usr`

Reply:
The temporary FTPServer defaults to writing certs and data there.

Reply:
I think `./deps/usr` is more typically used.

Reply:
👌

Reply:
I think you forgot to update this `.gitignore` rule